lib/install/config/loaders/installers/vue.js

```diff
@@ -1,13 +1,36 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const { env } = require('../configuration.js')
+
+// Change it to false if you prefer Vue styles to be inlined by javascript in runtime
+const extractStyles = true
+
+const cssLoader = [
+  { loader: 'css-loader', options: { minimize: env === 'production' } },
+  'postcss-loader'
+]
+const sassLoader = cssLoader.concat(['sass-loader?indentedSyntax'])
+const scssLoader = cssLoader.concat(['sass-loader'])
+
+function vueLoader(loader) {
```

Comment:
Seems like a rather confusing name as this is prepending the `vue-style-loader`, not `vue-loader`.